components/rss/sources/new-item-from-multiple-feeds/new-item-from-multiple-feeds.mjs

```diff
@@ -0,0 +1,38 @@
+import rssApp from "../../rss.app.mjs";
+
+export default {
+  key: "rss-new-item-from-multiple-feeds",
+  name: "New item from multiple RSS feeds",
+  type: "source",
+  description: "Emit new items from multiple RSS feeds.",
+  version: "0.0.1",
+  props: {
+    rssApp,
+    urls: {
+      type: "string[]",
+      label: "Feed URLs",
+      description: "Enter either one or multiple URLs from any public RSS feed.\n\n**Example:** `[\"https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml\", \"https://www.theguardian.com/uk/technology/rss\"]`",
+    },
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  dedupe: "unique",
+  async run() {
+    for (const url of this.urls) {
+      const feed = await this.rssApp.fetchFeed(this, url);
+      const items = await this.rssApp.parseFeed(feed);
+      for (const item of items) {
+        const itemKey = this.rssApp.itemKey(item);
```

Comment:
Is there any case this `item` could be repeated? Maybe it's a good idea to use an `uuid` or maybe the current timestamp. Same for the other places you are using `object-hash`. If the idea is just to generate a unique `id`. Maybe we can do it using a faster solution than hashing an object?

Reply:
yes if the user fetch from two rss that provides the exactly same information, for example cnn:world cnn:ny, the object could be  identical, and we don't want to notify twice the same information, that's why object hashing was used on this scenario.

Reply:
Got it